tests/devices/_internal/test_file_parser.py

```diff
@@ -111,6 +123,43 @@ def test_none_if_not_found(self, tmp_path):
         assert read_device_files([tmp_path]).online_id is None
 
 
+class TestReadsJlinkData:
+    def test_reads_target_name(self, tmp_path):
+        board_name = "FRDM-K66F"
+        pathlib.Path(tmp_path, "Segger.html").write_text(build_segger_html())
+        file_contents = build_board_html(f"https://www.dummy.com/{board_name}")
+        pathlib.Path(tmp_path, "Board.html").write_text(file_contents)
+        online_id = read_device_files([tmp_path]).online_id
```

Comment:
Can we be consistent with the newline spacing? I usually write tests using the "Given, When, Then" style, leaving newlines between each section:
```
GIVEN: mocks, inputs, paths, etc

WHEN: run code under test

THEN: assert results are as expected
```

We do have a few preexisting inconsistencies, but I've been trying to tidy them up where I can. It is kind of a nit-pick but I find the tests a lot easier to read when we use line spacing to denote the different sections.